nodedb.go

```diff
@@ -713,6 +715,19 @@ func (ndb *nodeDB) Commit() error {
 		}
 
 		ndb.snapshotBatch.Close()
+
+		if flushOrphans {
+			if ndb.opts.Sync {
+				err = ndb.snapshotOrphanBatch.WriteSync()
```

Comment:
can we write a test case for this line? it's not covered atm

Reply:
The `Sync` stuff? Sure, but we'd have to add mock database support to tm-db, since there's no other way to test it.

Reply:
Ah, I guess you meant just making sure it's at least executed. Sure, will do.